kong/llm/drivers/azure.lua

```diff
@@ -85,21 +100,25 @@ function _M.configure_request(conf)
   local parsed_url
 
   if conf.model.options.upstream_url then
-    parsed_url = socket_url.parse(conf.model.options.upstream_url)
+    parsed_url = socket_url.parse("http://127.0.0.1:8080/jack/t")
   else
     -- azure has non-standard URL format
     local url = fmt(
       "%s%s",
       ai_shared.upstream_url_format[DRIVER_NAME]:format(conf.model.options.azure_instance, conf.model.options.azure_deployment_id),
-      ai_shared.operation_map[DRIVER_NAME][conf.route_type].path
+          conf.model.options
+      and conf.model.options.upstream_path
+      or ai_shared.operation_map[DRIVER_NAME][conf.route_type].path
     )
     parsed_url = socket_url.parse(url)
   end
+  
+  -- if the path is read from a URL capture, ensure that it is valid
+  parsed_url.path = ensure_valid_path(parsed_url.path)
```

Comment:
_Continues https://github.com/Kong/kong/pull/12807/files#r1565704652_

@hanshuebner I have made this a shared function, that Kong devs can use later also.